libs/sdk-core/src/breez_services.rs

```diff
@@ -236,6 +236,11 @@ impl BreezServices {
         &self,
         req: SendPaymentRequest,
     ) -> Result<SendPaymentResponse, SendPaymentError> {
+        if req.bolt11.is_empty() {
```

Comment:
Perhaps this empty string check could be in the `parse_invoice` function?

Reply:
Makes sense, done